tests/test_message.py

```diff
@@ -1,18 +1,19 @@
-import asynctest
+import pytest
 
 from opsdroid.core import OpsDroid
 from opsdroid.message import Message
 from opsdroid.connector import Connector
 from opsdroid.cli.start import configure_lang
 
+configure_lang({})
 
-class TestMessage(asynctest.TestCase):
-    """Test the old opsdroid message class."""
+@pytest.fixture
+def opsdroid():
+    with OpsDroid() as opsdroid:
+        yield opsdroid
 
-    async def setup(self):
-        configure_lang({})
-
-    async def test_message(self):
+@pytest.mark.asyncio
+async def test_message(opsdroid, message):
```

Comment:
We dont have a message fixture so we need to remove this one

```suggestion
async def test_message(opsdroid):
```